cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -52,6 +52,10 @@ type MasterConfiguration struct {
 
 	// APIServerCertSANs sets extra Subject Alternative Names for the API Server signing cert
 	APIServerCertSANs []string
+
+	// EtcdCertSANs sets extra Subject Alternative Names for the etcd server and peer signing certs
+	EtcdCertSANs []string
```

Comment:
It's possible to have them listen on different interfaces.
I guessed it would be okay to just have a union of all of the needed SANs for peers and the server, but I could be wrong.

We could add another config for this.
I'm thinking it would be best to just add `EtcdPeerCertSANs []string` and not do anything fancy.
This makes the config more verbose, but it's flexible.

Do you think this is correct?

Reply:
Renamed this to `EtcdServerCertSANs` and added `EtcdPeerCertSANs`